models/observationSubmissions.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
     submissionsUpdatedHistory: Array,
     entityTypeId: "ObjectId",
     entityType: String,
+    programId: "ObjectId",
```

Comment:
@aman-tunerlabs - Make this mandatory. Also, add an index to these new fields in DB via migrations